### ISSUE TITLE
Flatpickr time formatting issue

### ISSUE THREAD

**Issue by andypullen** [verbb/formie#956]
**Description**
Using 12-hour time formats, the AM/PM value is not correctly set. Instead, "A" is always stored, whether or not AM or PM is selected.

**Steps to reproduce**
1. Create a form with a date time field
2. Select one of the 12-hour formats in the appearance settings
3. Include the form and submit with a PM value
![Screen Shot 2022-06-27 at 16 28 49@2x](https://user-images.githubusercontent.com/9940032/176053880-d84aec2b-489d-4c32-bf10-2303c64d0231.jpg)

The value that gets stored in the submission data is the AM time.

### Form settings
Please pick from the options below.

- Multi-page form: Yes or No – either
- Submission Method: Ajax or Page Reload – either
- Client-side Validation: Yes or No – either
- Custom Form Templates: Yes or No – no

**Additional info**
- Plugin version: 1.6.1
- Craft version: 3.7.45.2
- Multi-site: Yes

**Additional context**

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.2](https://github.com/verbb/formie/releases/tag/1.6.2)
